devtools/docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+. /opt/conda/etc/profile.d/conda.sh
+conda activate base
+conda activate gmso-dev
+
+if [ "$@" == "jupyter" ]; then
+      jupyter notebook --no-browser --notebook-dir /home/anaconda/data --ip="0.0.0.0"
```

Comment:
Just tested, this line fails since we do not have `jupyter` installed as a dependency. i can go into a shell session no problem though. The easiest fix is to probably include a line in the dockerfile RUN command to install jupyter after we create the environment.